library/psa_crypto.c

```diff
@@ -5306,12 +5306,23 @@ static psa_status_t psa_tls12_prf_psk_to_ms_set_key(
      * uint16 with the value N, and the PSK itself.
      */
```

Comment:
You should update the comment with the forming of the premaster with salt as other_secret in DHE_PSK & RSA_PSK as described in section 3 & 4 of RFC 4279

Reply:
Yes, documentation is still TBD.

Reply:
Fixed